poll-webapp/src/main/webapp/vue-app/poll-extensions/components/CreatePollDrawer.vue

```diff
@@ -115,21 +95,28 @@ export default {
 
   data(){
     return {
+      MAX_LENGTH: 1000,
```

Comment:
addOption and removeOption useless to be removed. We should clean all useless methods